satpy/etc/enhancements/generic.yaml

```diff
@@ -783,3 +783,97 @@ enhancements:
           stretch: crude
           min_stretch: [0, 0, 0]
           max_stretch: [1, 1, 1]
+
+  ash_abi:
```

Comment:
I'd like all ABI specific enhancements to go in `enhancements/abi.yaml`.

Reply:
It's ready